test_huggingface_live.py

```
@@ -0,0 +1,313 @@
+    tests = [
+        ("LLM Inference API", test_llm_inference_api()),
+        ("LLM Streaming", test_llm_streaming()),
+        ("Embedder Remote API", test_embedder_remote()),
+    ]
+
+    results = []
+    for test_name, test_coro in tests:
+        try:
+            result = await test_coro
+            results.append((test_name, result))
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)